### DISCUSSION TITLE
Preview a Store Image

### DISCUSSION THREAD

**Discussion by jimbrend** [btcpayserver/btcpayserver#6182] [Ideas / Feature Requests]
before making a change, 

i would like to see after uploading how the thumbnail looks that is 500kb for the Store I create, 

For example,

![image](https://github.com/user-attachments/assets/902b9208-0c41-4832-9a26-3b53910601ac)

Here, it was not cropped exactly to the design I had envisioned, before clicking save, that would be cool.

Thank you!

**Comment by dstrukt** [comment]
We've discussed this before, and IIRC historically it's been a technical limitation, and still might be, cc @dennisreimann but agree, would be a nice feature, and functionality that we could carry to many other parts of the app.

**Comment by jimbrend** [comment]
if you could share a PR or at least a draft concept even with pseudocode and notes on what is needed, I'd like to try and contribute thank you @dstrukt

**Comment by pavlenex** [comment]
<img width="1195" alt="Screenshot 2025-05-05 at 11 11 38" src="https://github.com/user-attachments/assets/da8d4148-27aa-4dd3-843a-74c5c14f546f" />
This exist now.
